content/docs/concepts/performance-and-scalability/index.md

```diff
@@ -13,127 +13,104 @@ aliases:
 keywords: [performance,scalability,scale,benchmarks]
 ---
 
-We follow a four-pronged approach to Istio performance characterization, tracking and improvements:
+Istio makes it easy to create a network of deployed services with rich routing,
+ load balancing, service-to-service authentication, monitoring, and more. It
+ does so without any changes to the application code. Istio strives to provide
+ these benefits with minimal resource overhead. Istio also aims to support very
+ large meshes with high request rates while adding minimal latency.
 
-* Code level micro-benchmarks
+## Data plane performance
 
-* Synthetic end-to-end benchmarks across various scenarios
+Istio consists of components that run on the data plane and components that
```

Comment:
I think that should be "in", not "on". Maybe enumerate the components in parenthesis for either planes?